### PR TITLE
RIA-3585: ITHC environment set up for ia-home-office-integration-api

### PR DESCRIPTION
**Pull request by divisathyan**
RIA-3585: ITHC environment set up for ia-home-office-integration-api and ia-home-office-mock-api, Enable home-office-integration in ithc ia-case-api

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3585

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: k8s/ithc/common/ia/case-api.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ spec:
       ingressHost: ia-case-api-ithc.service.core-compute-ithc.internal
       environment:
         RESTART_ME: "Update due to vault Change"
+        IA_HOME_OFFICE_INTEGRATION_ENABLED: true
```
Comment (makowalski) [inline k8s/ithc/common/ia/case-api.yaml:27]: let's put a String as usual:
`IA_HOME_OFFICE_INTEGRATION_ENABLED: "true"`

File: k8s/ithc/common/ia/home-office-integration-api.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    name: ia-home-office-integration-api
+    version: 0.0.12
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/home-office-integration-api:latest
+      ingressHost: ia-home-office-integration-api-ithc.service.core-compute-ithc.internal
+      environment:
+        HOME_OFFICE_ENDPOINT: "https://cts-challenge.prp1.np.immigration-api.homeoffice.gov.uk"
```
Comment (makowalski) [inline k8s/ithc/common/ia/home-office-integration-api.yaml:26]: let's use mock api as for the first test, we know that Gareth is working to white list the cluster but it is not done yet

File: k8s/ithc/common/ia/case-api.yaml (hunk 1 above)
Comment (divisathyan) [reply k8s/ithc/common/ia/case-api.yaml:27]: changed to String

File: k8s/ithc/common/ia/home-office-integration-api.yaml (hunk 2 above)
Comment (divisathyan) [reply k8s/ithc/common/ia/home-office-integration-api.yaml:26]: changing to mock api in ithc

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**